Description: Remove Google Analytics and Google+ javascript.
It is not relevant for an offline and private installation.
Author: Simon Chopin <chopin.simon@gmail.com>
Forwarded: not-needed
Last-Update: 2012-04-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/en/_templates/layout.html
+++ b/doc/en/_templates/layout.html
@@ -2,17 +2,4 @@
 
 {% block footer %}
 {{ super() }}
-<script type="text/javascript">
-
-  var _gaq = _gaq || [];
-  _gaq.push(['_setAccount', 'UA-7597274-13']);
-  _gaq.push(['_trackPageview']);
-
-  (function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-  })();
-
-</script>
 {% endblock %}
